app/controllers/users_controller.rb

```diff
@@ -5,6 +5,17 @@ def new
 
   def create
     @user = User.new(user_params)
+    #where returns a collection vs find_by.
+    # if @user.save
+    #   session[:id] = @user.id
+    #   session[:user_id] = @user.id
+    #   flash[:notice] = "You are now logged in #{user_params[:name]}"
+    #   redirect_to "/profile"
+    # else
+    #   flash[:error] = @user.errors.full_messages.join(". ").to_s
+    #   redirect_to "/register"
+    # end
+
```

Comment:
My mentor and I went over this and found a better way. I wanted to leave it as is but had to change it to make some of the admin stuff work. Sorry for whoever wrote the code I had to change.